src/util/system.cpp

```diff
@@ -1128,10 +1128,22 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 void runCommand(const std::string& strCommand)
 {
     if (strCommand.empty()) return;
+    int nErr = 0;
 #ifndef WIN32
-    int nErr = ::system(strCommand.c_str());
+    nErr = ::system(strCommand.c_str());
 #else
-    int nErr = ::_wsystem(std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t>().from_bytes(strCommand).c_str());
+    STARTUPINFOA info;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&info, sizeof(info));
+    info.cb = sizeof(info);
+    ZeroMemory(&processInfo, sizeof(processInfo));
+    if (CreateProcessA(NULL, const_cast<char *>(strCommand.c_str()), NULL, NULL, FALSE, 0x08000000, NULL, NULL, &info, &processInfo)) {
```

Comment:
`CreateProcessA` seems to execute the program directly, rather than passing through the shell as expected for `-*notify`. This may break existing usage, especially because `.bat` files will fail.

Reply:
According to the [doc](https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-createprocessa), "The CreateProcess function creates a new process, which runs independently of the creating process.".

Reply:
Indeed, a workaround is to put `cmd.exe` at the command beginning as in #5834, but I don't know if this is a good way to do it..